src/components/StringFormatter/_mocks_/index.js

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
I'm wondering if it is worth having this file, if it's just `value: 'Value'` being saved? Could we simplify and remove the mock file?